pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -251,18 +252,22 @@ func (e *TokensController) syncServiceAccount() {
 		klog.V(4).Infof("syncServiceAccount(%s/%s), service account deleted, removing tokens", saInfo.namespace, saInfo.name)
 		sa = &v1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Namespace: saInfo.namespace, Name: saInfo.name, UID: saInfo.uid}}
 		retry, err = e.deleteTokens(sa)
-		if err != nil {
+		if err != nil && !isNamespaceTerminatingError(err) {
```

Comment:
I think it would be better to prevent creating these requests instead of simply suppressing the messages.   I would support having an namespace informer and lister.  I'm pretty sure @liggitt came around after discovering how many of these requests we make during e2e runs

Reply:
@deads2k sure, I can wire that... PR updated.